src/login/logind-session.c

```diff
@@ -1393,7 +1393,11 @@ int session_kill(Session *s, KillWhom whom, int signo) {
         if (!s->scope)
```

Comment:
Please also move this check into `KILL_ALL` case.

Reply:
And done.